bat/tests/build-multiple-delta-packs/run.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+@test "build delta-packs after build all for different versions" {
+  current_ver=$(get-current-version)
+  mixer-init-stripped-down "$current_ver" 10
+  sudo -E mixer build all --native
+  sudo mixer versions update
+  mixer-build-all --native
```

Comment:
`mixer-build-all` already has the `--native` flag, so there is no need to add it again.